src/lucky_flow.cr

```diff
@@ -15,7 +16,7 @@ class LuckyFlow
     setting base_uri : String
     setting retry_delay : Time::Span = 10.milliseconds
     setting stop_retrying_after : Time::Span = 1.second
-    setting chromedriver_path : String = LuckyFlow.default_driver_path
+    setting chromedriver_path : String?
```

Comment:
@paulcsmith This might be a bug with Habitat or a known limitation but when I had the installation here and caused a failure, it failed with a message about recursion while setting class variables.

Reply:
Hmm could you open an issue in Habitat? I don’t think I’ve seen that before and am not sure I understand what caused it but it seems like we should support it or at least raise a better error :D

Reply:
Here's the issue https://github.com/luckyframework/habitat/issues/46